app/models/prisoner.rb

```diff
@@ -5,12 +5,21 @@ class Prisoner
   attribute :number, String
   attribute :prison_name, String
 
-  validates_format_of :number, with: /\A[a-z]\d{4}[a-z]{2}\z/i, message: "must be a valid prisoner number" # eg a1234aa
-  validates_inclusion_of :prison_name, in: Rails.configuration.prison_data.keys, message: "must be chosen"
+  validates :number, format: {
+    with: /\A[a-z]\d{4}[a-z]{2}\z/i,
+    message: "must be a valid prisoner number" # eg a1234aa
+  }
+
+  validates :prison_name, inclusion: {
+    in: Rails.configuration.prison_data.keys,
+    message: "must be chosen"
+  }
+
   validate :prison_in_service
 
   def prison_in_service
-    if !self.prison_name.blank? && !Rails.configuration.prison_data[self.prison_name]['enabled']
+    if !self.prison_name.blank? &&
+      !Rails.configuration.prison_data[self.prison_name]['enabled']
```

Comment:
This could be shorter without the redundant `self`:

```ruby
if prison_name.present? && !Rails.configuration.prison_data[prison_name]['enabled']
```